shared/vcjs/verifyCredential.ts

```diff
@@ -70,17 +71,35 @@ export async function verifyCredential(
   }
 }
 
-function handleResponse(result: any) {
+function handleResponse(
+  result: any,
+  verifiableCredential: VerifiableCredential | Credential,
+) {
   var errorMessage = VerificationErrorType.NO_ERROR;
   var isVerifiedFlag = true;
 
   if (!result?.verified) {
-    if (result['results'][0].error.name == 'jsonld.InvalidUrl') {
+    let errorCodeName = result['results'][0].error.name;
+    if (errorCodeName == 'jsonld.InvalidUrl') {
       errorMessage = VerificationErrorType.NETWORK_ERROR;
+      isVerifiedFlag = false;
+    } else if (errorCodeName == VerificationErrorType.RANGE_ERROR) {
+      errorMessage = VerificationErrorType.RANGE_ERROR;
+      const vcIdentifier = verifiableCredential.credentialSubject.UIN
```

Comment:
can we add a utility method which returns isUIN -> true or false and simplify the logic here?

Reply:
Extracted a utility Method.